src/components/experiments/single-view/overview/VariationsTable.tsx

```diff
@@ -71,30 +72,21 @@ function assignmentHref(variationName: string, experimentName: string) {
       case 'user_not_assignable':
         alert('You must be proxied or sandboxed to use this bookmark');
         break;
+      case 'invalid_ids':
+        alert('To assign yourself, you must run this from Abacus.\\n\\nTo assign the current anonymous user, verify ' +
+              'that Tracks is not blocked and run in an environment where the tk_ai cookie is set.');
+        break;
       default:
         if (!responseBody.variations) {
           alert('An unknown error occurred: ' + responseBody.message);
           break;
         }
-        const duration = responseBody.duration === 'unlimited' ?
-          responseBody.duration : Math.ceil(responseBody.duration / 60 / 60);
-
-        if (responseBody.storage_method === 'cookie') {
-          window.localStorage.setItem(
-            'explat-experiment--${experimentName}',
-            JSON.stringify(
-              {
-                'experimentName':'${experimentName}',
-                'variationName': '${variationName}',
-                'retrievedTimestamp': Date.now(),
-                'ttl': responseBody.duration === 'unlimited' ? Infinity : responseBody.duration,
-              }
-            )
-          );
-
-          alert('ExPlat: Successful Assignment\\n–––––––––––––––––––––––––––––\\n\\nExperiment: ${experimentName}\\nVariation: ${variationName}\\n\\nMethod: Logged-out assignment, expires in ' + duration + ' hours\\nClient-side: Applies to the current domain (LocalStorage).\\nServer-side: Applies to current session (Cookie).');
+        const baseMessage = 'ExPlat: Successful Assignment\\n–––––––––––––––––––––––––––––\\n\\n' +
+          'Experiment: ${experimentName}\\nVariation: ${variationName}\\n\\n';
+        if (responseBody.storage_method === 'anon_sqooped_out_table') {
+          alert(baseMessage + 'Method: Logged-out assignment\\nApplies to the current anon user.');
```

Comment:
```suggestion
          alert(baseMessage + 'Method: Logged-out assignment\\nApplies to the current anon user (tk_ai).');
```
I think mentioning the mechanism is useful as otherwise it can seem a bit black boxy.

Reply:
Good point :slightly_smiling_face: